app/mailers/submit_application_reminder_mailer.rb

```diff
@@ -1,8 +1,14 @@
-class SubmitApplicationReminderMailer < GovukNotifyRails::Mailer
+class SubmitApplicationReminderMailer < BaseApplyMailer
   self.delivery_job = GovukNotifyMailerJob
   require_relative 'concerns/notify_template_methods'
   include NotifyTemplateMethods
 
+  INELIGIBLE_STATES = %w[use_ccms generating_reports submitting_assessment assessment_submitted].freeze
```

Comment:
generating_reports makes use of PostSubmissionProcessingJob to cancel its jobs so is there a duplication here?